src/local-time.lisp

```diff
@@ -1738,16 +1738,16 @@ See the documentation of FORMAT-TIMESTRING for the structure of FORMAT."
             (cond
               ((member fmt '(:gmt-offset :gmt-offset-or-z :gmt-offset-hhmm))
                (multiple-value-bind (offset-hours offset-secs)
-                   (floor offset +seconds-per-hour+)
+                   (truncate offset +seconds-per-hour+)
                  (declare (fixnum offset-hours offset-secs))
                  (if (and (eql fmt :gmt-offset-or-z) (zerop offset))
                      (princ #\Z result)
                      (format result "~c~2,'0d~:[:~;~]~2,'0d"
-                             (if (minusp offset-hours) #\- #\+)
+                             (if (minusp offset) #\- #\+)
                              (abs offset-hours)
                              (eql fmt :gmt-offset-hhmm)
-                             (truncate (abs offset-secs)
-                                       +seconds-per-minute+)))))
+                             (round (abs offset-secs)
```

Comment:
I'm concerned this will change output in a breaking manner.

Reply:
It will, although only for offsets that do not even align on minutes.
I'll revert.